ui/apps/platform/cypress/integration/vulnmanagement/reporting/reportingForm.test.js

```diff
@@ -123,4 +130,112 @@ describe('Vulnerability Management Reporting form', () => {
         // TODO: once we are able to manipulate the PatternFly select element, uncomment and complete the test
         // cy.get('button:contains("Create")').should('be.enabled').click();
     });
+
+    it('should allow creation of a report configuration', () => {
+        if (!isCollectionsEnabled) {
```

Comment:
Instead of adding special cases for the current access scope implementation, we'll just ignore this test until collections goes in.